tests/detect_machine.sh

```diff
@@ -86,9 +86,15 @@ elif [[ -d /lfs/h1 && ! -d /lfs/h3 ]]; then
 elif [[ -d /mnt/lfs1 ]]; then
   # We are on NOAA Jet
   MACHINE_ID=jet
-elif [[ -d /scratch1 ]]; then
-  # We are on NOAA Hera
-  MACHINE_ID=hera
+elif [[ -d /scratch3 ]]; then
+  # We are on NOAA Hera or Ursa
+  mount=$(findmnt -n -o SOURCE /home)
```

Comment:
I should have tested this on Hera, but I did not.  `/home` is not mounted to a drive with `hera` in the name on Hera, so it will not work there.  However, `/apps` is (and this is also true for Ursa).  Can you please replace this with the following tested solution:

```suggestion
  mount=$(findmnt -n -o SOURCE /apps)
```

Reply:
@DavidHuber-NOAA I have tested your comments on Hera and Ursa, and the build results look good to me. Since @BenjaminBlake-NOAA is on leave, I will go commit this change.